src/mme-app/utils/mmeCommonUtils.cpp

```diff
@@ -192,25 +192,26 @@ SM::ControlBlock* MmeCommonUtils::findControlBlock(cmn::utils::MsgBuffer* buf)
 				int cbIndex = SubsDataGroupManager::Instance()->findCBWithimsi(IMSIInfo);
 				if (cbIndex > 0)
 				{
-                    log_msg(LOG_DEBUG, "existing cb for IMSI.\n");
+					log_msg(LOG_DEBUG, "existing cb for IMSI.\n");
 					cb = SubsDataGroupManager::Instance()->findControlBlock(cbIndex);
 				}
-				
-                if (cb == NULL)
+				if (cb == NULL)
 				{
-                    log_msg(LOG_DEBUG, "create new cb for IMSI.\n");
+					log_msg(LOG_DEBUG, "create new cb for IMSI.\n");
```

Comment:
would be good to print imsi in this log since it allocates CB first time for imsi here. Also we should try to print it with readable number.

Reply:
Taken care.